README.md

```diff
@@ -87,7 +87,7 @@ Find the list at:
 
 ### Community Calendar
 
-We have a community [Google Calendar](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84@group.calendar.google.com&ctz=UTC) you can watch for events and sync to. We will add relevant events concerning IPFS to it, both with geographical and interplanetary (online) locations. 
+We have a community on [Google Calendar](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84@group.calendar.google.com&ctz=UTC), where you can watch for events and sync to. We will add relevant events concerning IPFS to it, both with geographical and interplanetary (online) locations. 
```

Comment:
@TheBinitGhimire It's our community's Google Calendar, as opposed to our having a community on Google Calendar, so please remove the "on". Otherwise all else looks good!

Reply:
Hello @jessicaschilling,

Thank you for letting me know about it! I have made the changes as you mentioned.

Good to see that you have approved the changes!

Thanks,
Binit Ghimire